cmd/server/app/options/options.go

```diff
@@ -77,6 +78,13 @@ type ProxyRunOptions struct {
 	// blocking call has its own problems, so it cannot easily be made race condition safe.
 	// The check is an "unlocked" read but is still use at your own peril.
 	WarnOnChannelLimit bool
+
+	// Cipher suites used by the server.
+	// If empty, the default suite will be used from tls.CipherSuites(),
+	// also checks if given comma separated list contains cipher from tls.InsecureCipherSuites().
+	// NOTE that cipher suites are not configurable for TLS1.3,
+	// see: https://pkg.go.dev/crypto/tls#Config, so in that case, this option won't have any effect. 
```

Comment:
Maybe we could add the TLS version to the configuration also. Otherwise, user would not know what TLS version is used in the code. Default would TLS1.2, but would be possible to set it to TLS1.3. What do you think? Thank you!

Reply:
TLS version is the min version that server could accept. But you are right, for TLS1.3 it won't have any effect, so I added this info to the option arguments. Thank you!